### PR TITLE
Part0 exercise

### PR DESCRIPTION
**Pull request by Tanmay-say**
Sir , i have created three branches for that main , develop , courseinfo -> feature branch , But in this the develop branch is missing . The direct option the main <- courseinfo is tobe seem .

### CODE REVIEW

File: part0/Ex_0_4_New_note_diagram.mmd [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    deactivate server
+
+    Note right of browser: The browser starts executing the JavaScript code that fetches the JSON from the server
+
+    browser->>server: GET https://studies.cs.helsinki.fi/exampleapp/data.json
+    activate server
+    server-->>browser: [{ "content": "HTML is easy", "date": "2023-1-1" }, ... ]
+    deactivate server
+
+    Note right of browser: The browser executes the callback function that renders the notes
```
Comment (mavenik) [inline part0/Ex_0_4_New_note_diagram.mmd:27]: @Tanmay-say Where is the new note being created in this flow?

**Review by mavenik — CHANGES_REQUESTED** [review]
Except for 0.4, the rest looks okay to me

File: part0/Ex_0_4_New_note_diagram.mmd (hunk 1 above)
Comment (Tanmay-say) [reply part0/Ex_0_4_New_note_diagram.mmd:27]: okh sir  , i make the Changes there .

**Comment by Tanmay-say** [comment]
Sir , I have modified all the changes , please review it .

**Review by mavenik — CHANGES_REQUESTED** [review]
Looks good to me. I just noticed that the PR is to merge your code directly into `main` branch. Any reason why we aren't following git-flow branching model here?

**Comment by Tanmay-say** [comment]
Sir , i have create the develop branch also but it not giving me the
develop  branch option .

**Comment by Tanmay-say** [comment]
Sir , i have created the develop branch and I have push all the  code on the courseinfo branch but , there is not showing the develop branch option on my local machine .... Can we discuss this on today's meet .

**Pull request merged**